04/modules/WeatherGUI.js

```diff
@@ -0,0 +1,167 @@
+class WeatherGUI {
+    constructor(api) {
+        this.api = api;
+        this.formEl = document.querySelector('.form');
+        this.weatherInfoElement = document.querySelector('.weather');
+    }
+
+    load() {
```

Comment:
Może zamiast `load()` - bo niby co wczytujesz? Dałbym `run()`, `start()` lub `init`, ale wtedy funkcja wew. to `initEvents()`

Reply:
Racja, dużo przyjemniej czyta się kod :).